scripts/reconcile

```diff
@@ -23,22 +23,27 @@ PACKAGES=$(osc se --package cri-o)
 PROJECT_PREFIX=isv:kubernetes:addons:cri-o
 PROJECT_PRERELEASE_PREFIX=$PROJECT_PREFIX:prerelease
 BUILD_PROJECT_SUFFIX=:build/cri-o
+BRANCH_MAIN=main
 
-ACTIVE_VERSIONS=$(curl_retry https://raw.githubusercontent.com/cri-o/cri-o/refs/heads/main/internal/version/version.go | sed -En "s/.*ReleaseMinorVersions.*\{(.*)\}/\1/p")
+VERSION_URL=https://raw.githubusercontent.com/cri-o/cri-o/refs/heads/$BRANCH_MAIN/internal/version/version.go
```

Comment:
```suggestion
VERSION_URL=https://raw.githubusercontent.com/cri-o/cri-o/refs/heads/${BRANCH_MAIN}/internal/version/version.go
```

Reply:
The line wrapping is horrible here for some reason.

Reply:
Fixed